### ISSUE TITLE
[Bug]: getUrl is requesting the API

### ISSUE THREAD

**Issue by simonmilz** [codebar-ag/laravel-flysystem-cloudinary#48]
### What happened?

I am using the Cloudinary Nova Field. After I added the CloudinaryImage Field to my resource the UI was getting really slow. While I was investigating for a while, I found out that https://github.com/codebar-ag/laravel-flysystem-cloudinary/blob/main/src/FlysystemCloudinaryAdapter.php#L523 is calling the API, instead of simply returning the public URL:

```
try {
    $response = $this->explicit($path);
} catch (NotFound) {
    return false;
}
```

IMO this would be enough and MUCH faster:
```
try {
    return (string) $this->cloudinary->image($path)->toUrl();
} catch (NotFound) {
     return false;
}
```

### Package Version

3.1

### PHP Version

8.2

### Laravel Version

10.0

**Comment by StanBarrows** [comment]
Hey @simonmilz 

Many thanks for the hint. We'll fix this within this week.

Best regards
Sebastian

**Comment by simonmilz** [comment]
For the Nova Field you should consider to use the `preview` and `thumbnail` methods to generate smaller images: https://github.com/Silvanite/nova-field-cloudinary/blob/master/src-php/Fields/CloudinaryImage.php#L24 

This would improve the field even more ;)

**Comment by RhysLees** [comment]
@simonmilz I made a pr #49 for this, feel free to take a look and let me know if it checks out for you

**Comment by StanBarrows** [comment]
@RhysLees thanks for the updating package.
 
@simonmilz 
- The Nova improvements are available with >= v2.2  (laravel-flysystem-cloudinary-nova) 
- This package depends on >= v4.0  laravel-flysystem-cloudinary pacakge.

Please note that both versions are currently in alpha release. We await the Laravel 11 Release to mark both version bumps as stable. The stable release is expected in the first week of March.
